src/main/java/org/minions/devfund/chelo/Actor.java

```diff
@@ -0,0 +1,72 @@
+package org.minions.devfund.chelo;
+
+import java.util.ArrayList;
+
+/**
+ * Actor class.
+ */
+public class Actor {
+    private String name;
+    private ArrayList<Movie> movies;
+
+    /**
+     * Actor constructor.
+     * @param name actor
+     */
+    public Actor(final String name) {
+        this.name = name;
+        movies = new ArrayList<>();
+    }
+
+    /**
+     * Actor constructor.
+     */
+    public Actor() {
+        this("");
+    }
+
+    /**
+     * A method to get Actor's name.
+     * @return name
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * A method to set Actor's name.
+     * @param name set name.
+     */
+    public void setName(final String name) {
+        this.name = name;
+    }
+
+    /**
+     * A method to get Actor's movies.
+     * @return movies list.
+     */
+    public ArrayList<Movie> getMovies() {
+        return movies;
+    }
+
+    /**
+     * A method to set Actor's movies.
+     * @param movies actor's movies
+     */
+    public void setMovies(final ArrayList<Movie> movies) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "movies" object should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)